internal/ecrconsumer/sqs.go

```diff
@@ -0,0 +1,67 @@
+package ecrconsumer
+
+import (
+	"context"
+	"time"
+
+	"github.com/Wattpad/sqsconsumer"
+	"github.com/Wattpad/sqsconsumer/middleware"
+
+	// Metric Imports
+	kitlog "github.com/go-kit/kit/log"
```

Comment:
We can omit the qualified import, since the stdlib `log` package isn't in scope.